src/Bunny/AbstractClient.php

```diff
@@ -66,6 +66,12 @@
     /** @var int */
     protected $frameMax = 0xFFFF;
 
+    /** @var int  */
+    protected $nextChannelId = 1;
+
+    /** @var int  */
+    protected $channelMax = 255;
```

Comment:
I haven't found this limit anywhere in the spec - if there is no limit, channel max should be `0xFFFF` as channel ID is 16bit number.